vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.props" Condition="Exists('..\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.props')" />
```

Comment:
>Import [](start = 3, length = 6)

Please move the new imports down where we import the C++/WinRT props file.
Having it here prevents us from using the $(SolutionDir) variable.

Reply:
The same is for other project files.

---
In reply to: [456013669](https://github.com/microsoft/react-native-windows/pull/5533#discussion_r456013669) [](ancestors = 456013669)

Reply:
yea, did so as part of solutiondir change.